pulp_2to3_migration/app/migrations/0006_auto_20200318_2051.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 2.2.11 on 2020-03-18 20:51
+
+from django.db import migrations, models
```

Comment:
Seems like there are a bunch of built up missing modelling changes lacking migrations.  Is this intentional?

Reply:
I'm not sure what you mean and I don't see any migration in this PR at the moment.

Reply:
I have the file locally. The changes are all in the docker plugin